### ISSUE TITLE
JOSS Review: No output file defined in stochastic simulation

### ISSUE THREAD

**Issue by ayush9pandey** [copasi/basico#35]
In `Creating_a_simple_model.ipynb` on binder:
```
result = run_time_course(duration=50, method='stochastic')

Error while initializing the simulation: >ERROR 2023-07-06T07:33:31<
  CCopasiTask (5): No output file defined for report of task 'Time-Course'.
```

**Comment by fbergmann** [comment]
What the example was trying to show, was a common pitfall with running stochastic simulations with COPASI, that they will fail if there are too many particles to be simulated, so the failure is expected, and explained in the notebook: 

![notebook](https://p194.p3.n0.cdn.getcloudapp.com/items/X6u7AAQy/da01a50d-47cb-4e92-bb64-5a7170ff0388.jpg?v=929d7b6a191b837665a0f4824cedb75c)


I can see that the error message is confusing though, so I should try and do something about that.

**Comment by ayush9pandey** [comment]
I see! I was confused by "output file not defined" message, which seemed like a syntactic error.

**Comment by fbergmann** [comment]
This has been released with release 0.51. Thank you again
